src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -0,0 +1,130 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('material.components.fabSpeedDial', [])
+    .directive('mdFabSpeedDial', MdFabSpeedDialDirective);
```

Comment:
I am not sure what is the convention in the `ngMaterial` codebase (and feeling to lazy to go checkit out right now), but I prefer to have non-instantiable functions start with a lowercase.